pyrightconfig.stricter.json

```
@@ -72,7 +72,8 @@
         "stubs/tqdm",
         "stubs/ttkthemes",
         "stubs/urllib3",
-        "stubs/vobject",
+        "stubs/vobject",,
+        "stubs/whichcraft"
```

Our script has a bug that causes a syntax error when adding to the end of the script. It might be easy to fix, if you're interested :)

But that said, I think we don't need to add this to the stricter exclude list at all:

```suggestion
        "stubs/vobject",
```